src/test/java/net/sf/jabref/importer/fileformat/JSONEntryParserTest.java

```diff
@@ -28,8 +28,7 @@
     private final JSONEntryParser jc = new JSONEntryParser();
     @Test
     public void testBibJSONConverter() {
-        String jsonString = new String(
-                "{\n\"title\": \"Design of Finite Word Length Linear-Phase FIR Filters in the Logarithmic Number System Domain\",\n"
+        String jsonString = "{\n\"title\": \"Design of Finite Word Length Linear-Phase FIR Filters in the Logarithmic Number System Domain\",\n"
```

Comment:
In https://github.com/JabRef/jabref/pull/378 we work out how to read files during test. I put the source files (`.xml`; `.json` in your case) and target files (`.bib`) into version control and the tests read the files and compare them. This makes maintenance IMHO easier. Furthermore, you can test all fields by one call and don't need to add `assertEquals` for each field.